plugins/intel-usb4/README.md

```diff
@@ -0,0 +1,32 @@
+# Intel USB4
+
+## Introduction
+
+USB-4 is a hardware interface specification. This plugin supports the Intel Goshen Ridge hardware.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+an unspecified binary file format, with vendor specific header.
+
+This plugin supports the following protocol ID:
+
+* com.intel.thunderbolt
+
+## GUID Generation
+
+These devices use a custom generation scheme:
+
+* `TBT-00d4b071`
```

Comment:
I think you need to reference that this is identical to thunderbolt plugin; quite intentionally.

Reply:
Done.